src/main/java/org/jmhsrobotics/frc2024/utils/ControllerRumble.java

```diff
@@ -0,0 +1,42 @@
+package org.jmhsrobotics.frc2024.utils;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
+public class ControllerRumble {
+  private GenericHID controller;
+
```

Comment:
If you want this class to be a command. It must extend "Command"
For more information I recommend reading the following documentation
https://docs.wpilib.org/en/stable/docs/software/commandbased/index.html